backend/uclapi/dashboard/models.py

```diff
@@ -38,6 +40,30 @@ class App(models.Model):
     created = models.DateTimeField(auto_now=False, auto_now_add=True)
     last_updated = models.DateTimeField(auto_now=True)
 
+    client_id = models.CharField(
```

Comment:
should we create these OAuths specific fields on the App model? I think I'd prefer if they were in a separate model with a OneToOne relationship to `App`

Reply:
Good question. On one hand, I'm okay with doing this because they're fields that are atomic to apps, but on the other hand splitting off OAuth data seems fair too. Depends how separate you see OAuth to the rest of the project.

Reply:
I think I'd prefer if they were separate purely for flexibility reasons. That's the reason I did it in webhooks if you want to check that out :)

Reply:
Doable: yes
Absolute pain that probably requires wiping out the data in staging and reprogramming the migrations? Also yes.

Reply:
¯\_(ツ)_/¯ technical debt
It's definitely easier to change it now than later, but up to you

Reply:
Sort of technical debt and sort of isn't. Not having an unnecessary join when every app will have all that data is a good thing. It's a question of balancing DB operations in an app we hope will scale vs. code clarity and avoiding ugly monoliths. I have no particular affinity with either side of this; however, I veer toward leaving it the same because of the effort involved in changing it lol